lib/core/src/Cardano/Wallet/DB/Sqlite/Types.hs

```diff
@@ -263,6 +267,35 @@ instance PathPiece TxId where
     toPathPiece = toText . getTxId
     fromPathPiece = fmap TxId . fromTextMaybe
 
+--------------------------------------------------------------------------------
+-- Tokens
+--------------------------------------------------------------------------------
+
+instance PersistField TokenName where
+    toPersistValue = toPersistValue . toText
+    fromPersistValue = fromPersistValueFromText
+
+instance PersistFieldSql TokenName where
+    sqlType _ = sqlType (Proxy @Text)
+
+instance PersistField TokenPolicyId where
+    toPersistValue = toPersistValue . toText
+    fromPersistValue = fromPersistValueFromText
+
+instance PersistFieldSql TokenPolicyId where
+    sqlType _ = sqlType (Proxy @Text)
+
+instance PersistField TokenQuantity where
+    -- The `persistent-sqlite` package has no big number type, so we use a
```

Comment:
@rvl wrote:
> It would be more correct to say that sqlite has no big integer type.

Good point!

I've updated the comment to reflect this.

Fixed in db9c2658d97c82254e6b561802e91251f1674db0.

Reply:
Thanks